Resources/views/fields/ezbinaryfile_video.html.twig

```diff
@@ -3,9 +3,8 @@
 
 {# BinaryFile that contains a video and that we want to display as HTML5 video. #}
 {% block ezbinaryfile_field %}
-    {% set videoUri = '/content/download/' ~ contentInfo.id ~ '/' ~ field.id
-                        ~ '/version/' ~ contentInfo.currentVersionNo ~  "/file/"
-                        ~ field.value.fileName|escape( 'url' ) %}
+    {% set videoUri = '/content/download/' ~ contentInfo.id ~
```

Comment:
You should use the route reference for this, like it is done in the default field template.